docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_landing_page_level_2.yml

```diff
@@ -84,6 +84,5 @@ migration_dependencies:
     optional:
         - yukon_migrate_icon
         - yukon_migrate_primary_content
-        - yukon_migrate_secondary_content
```

Comment:
Does deleting solve the issue?
Shouldn't it be required instead of optional?

Reply:
Yes. We removed it because there is no field for secondary content on **Landing Page Level 2**.  

Screeshot attached.

![landing_page_level_2](https://github.com/ytgov/yukon-ca/assets/150289876/7242b72c-2ae4-4be4-a9dd-6d5b07efb237)